File: csharp/DVSClient/DVSClient/server/phone/RestApiPhoneValidateResponse.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Phone
{
    public class RestApiPhoneValidateResponse : RestApiResponse
    {
        [JsonProperty("error")]
        public RestApiResponseError? Error { get; set; }

        [JsonProperty("result")]
        public RestApiPhoneValidateResult? Result { get; set; }

        [JsonProperty("metadata")]
        public RestApiPhoneValidateMetadata? Metadata { get; set; }
    }
}

File: typescript/src/server/phone/restApiPhoneValidateResponse.ts
import { RestApiResponse } from "../restApiResponse";
import { RestApiResponseError } from "../restApiResponseError";
import { RestApiPhoneValidateMetadata } from "./restApiPhoneValidateMetadata";
import { RestApiPhoneValidateResult } from "./restApiPhoneValidateResult";

export type RestApiPhoneValidateResponse = RestApiResponse & {
    error?: RestApiResponseError;
    result?: RestApiPhoneValidateResult;
    metadata?: RestApiPhoneValidateMetadata;
};